src/test/java/org/prebid/server/bidder/gothamads/GothamAdsBidderTest.java

```diff
@@ -0,0 +1,246 @@
+        final Bid videoBid = Bid.builder().impid("2").mtype(2).build();
+        final Bid nativeBid = Bid.builder().impid("3").mtype(4).build();
+
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                givenBidRequest(identity()),
+                givenBidResponse(bannerBid, videoBid, nativeBid)
+        );
+
+        // when
+        final Result<List<BidderBid>> result = sut.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).containsExactly(
+                BidderBid.of(bannerBid, banner, null),
+                BidderBid.of(videoBid, video, null),
+                BidderBid.of(nativeBid, xNative, null)
+        );
+    }
+
+    @Test
+    public void shouldReturnErrorWhileMakingBidsWhenImpTypeIsNotSupported() {
+        // given
+        final Bid audioBid = Bid.builder().impid("3").mtype(3).build();
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                givenBidRequest(identity()),
+                givenBidResponse(audioBid)
+        );
+
+        // when
+        final Result<List<BidderBid>> result = sut.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors())
+                .containsExactly(badServerResponse("Unable to fetch mediaType 3 in multi-format: 3"));
+    }
+
+    private static Stream<Arguments> responsesWithoutSeatBids() {
+        return Stream.of(
+                Arguments.of(givenEmptyBidResponse()),
+                Arguments.of(givenBidResponse())
+        );
+    }
+
+    private static Result<List<HttpRequest<BidRequest>>> expectedResults(BidRequest bidRequest, String... impIds) {
+        return Result.withValue(HttpRequest.<BidRequest>builder()
+                .method(HttpMethod.POST)
+                .uri("https://test-url.com/?pass=accountId")
+                .headers(expectedHeaders())
+                .impIds(impIds.length == 0 ? Set.of(IMP_ID) : Set.of(impIds))
+                .body(jacksonMapper.encodeToBytes(bidRequest))
+                .payload(bidRequest)
+                .build()
+        );
+    }
+
+    private static MultiMap expectedHeaders() {
+        return headers()
+                .set(CONTENT_TYPE_HEADER, APPLICATION_JSON_CONTENT_TYPE)
+                .set(ACCEPT_HEADER, APPLICATION_JSON_VALUE)
+                .set(X_OPENRTB_VERSION_HEADER, OPEN_RTB_VERSION)
+                .set(USER_AGENT_HEADER, DEVICE_UA)
+                .set(X_FORWARDED_FOR_HEADER, List.of(IPV_6_HEADER_VALUE, IP_HEADER_VALUE));
+    }
+
+    private static BidRequest givenBidRequest(Function<Imp.ImpBuilder, Imp.ImpBuilder>... impCustomizers) {
+        return givenBidRequest(identity(), List.of(impCustomizers));
+    }
+
+    private static BidRequest givenBidRequest(
+            Function<BidRequest.BidRequestBuilder, BidRequest.BidRequestBuilder> bidRequestCustomizer,
+            List<Function<Imp.ImpBuilder, Imp.ImpBuilder>> impCustomizers) {
```

Comment:
Please, place new line after method signature with multi-line argument declarations. This is also a part of our internal code-style. Also, please, check for similar occurences.